src/pubtools/_marketplacesvm/cloud_providers/ms_azure.py

```diff
@@ -351,11 +351,11 @@ def _delete_push_images(self, push_item, **kwargs):
             builds (List[str])
                 The builds to delete.
         Returns:
-            A VHDPushItem
+            A tuple of VHDPushItem and False at the moment.
         """
         # TODO: Add delete functionality to Azure
         LOG.info("Deleting of Azure images from a push is not implemented yet")
-        return push_item
+        return push_item, False
```

Comment:
I would return None for empty extra "object" from the method
But False is fine as well

Reply:
Fixed